core/server/webhooks-events.js

```diff
@@ -0,0 +1,79 @@
+var _ = require('lodash'),
```

Comment:
Is this the best location for this file? I followed the `analytics-events.js` style because I couldn't see anywhere else we have a general events listener.

Also, where would the tests for this go?

Reply:
According to the new world order: https://github.com/TryGhost/Ghost/issues/9178 

I would put this in `services/webhooks.js` and the tests in `services/webhooks_spec.js`